src/ingest.py

```diff
@@ -233,6 +254,22 @@ def main():
                             info = Info.from_path(resource.geturl())
                             r.lrem(worker_queue, 0, info.path)
 
+                if is_lfa and resources_photodiode:
```

Comment:
Since this is the same structure as the previous section, it might be nice to abstract that out into a function, in case we have more ingesters later.

Reply:
I'm also bothered by the repeated structure and want to abstract it out.   But as the camera team wants this asap I'll defer the refactoring to a future ticket.